src/healthkit.cs

```diff
@@ -586,6 +602,31 @@ interface HKMetadata {
 		[Watch (3,0), iOS (10,0)]
 		[Export ("SwimmingStrokeStyle")]
 		NSString SwimmingStrokeStyle { get; }
+
+		//FIXME: Types for the following properties:
+		[Watch (4, 0), iOS (11, 0)]
+		[Export ("SyncIdentifier")]
+		string SyncIdentifier { get; }
+
+		[Watch (4, 0), iOS (11, 0)]
+		[Export ("SyncVersion")]
+		NSNumber SyncVersion { get; }
```

Comment:
Can we make this a more strong type? not sure if it is documented what actual number is in here

Reply:
I couldn’t find any specific info on what kind of number this gives. I’ll see if they have a sample using it to see what it gives.